.tekton/basic-component-pipeline.yaml

```diff
@@ -206,10 +206,10 @@ spec:
     taskRef:
       name: output-image-tag-construction
```

Comment:
Now there's no reason for `basic-component-pipeline` to depend on or include the `output-image-tag-construction`. Could you please delete the entire `output-image-tag-construction` task from this pipeline and adjust result references from `tasks.output-image-tag-construction.results.FINAL_IMAGE_TAG` to `tasks.determine-image-tag.results.IMAGE_TAG`?

Reply:
This one is also still open.